.circleci/config.yml

```diff
@@ -475,30 +488,11 @@ jobs:
 
 workflows:
   version: 2
-  operator:
```

Comment:
Do not remove operator workflow

Reply:
If you see the commit title related to that, it is just for testing purposes

Reply:
Remove the commit, open new branch out of this branch, apply the commit and test it.

Reply:
It's tested  https://circleci.com/workflow-run/0775956f-4233-4cf8-9f0f-69ba43408369  (nightly only) and current state is without the nightly execution